chain/rust/src/builders/utils.rs

```diff
@@ -99,22 +99,24 @@
 //     }
 // }
 
-#[cfg(test)]
-mod tests {
-    use super::*;
+// #[cfg(test)]
```

Comment:
why commented?

Reply:
I think I shall just remove this entire file instead.